releasetool/commands/tag/nodejs.py

```diff
@@ -111,7 +111,7 @@ def create_release(ctx: TagContext) -> None:
     click.secho("> Creating the release.")
 
     if ctx.upstream_repo in manifest_release:
-        # delegate releaase tagging to release-please
+        # delegate release tagging to release-please
         default_branch = ctx.release_pr["base"]["ref"]
         repo = ctx.release_pr["base"]["repo"]["full_name"]
```

Comment:
I believe we'll need to pull this above the `if` so that it can be used in the other block.